apps/academy/src/components/ConnectButton.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable @typescript-eslint/strict-boolean-expressions */
+import { ConnectButton as RainbowkitConnectButton } from "@rainbow-me/rainbowkit";
+import Image from "next/image";
+import { Button } from "ui/components/ui/button";
+
```

Comment:
@peteruche21 when the user clicks on the `Connect` button, the verify your account modal looks translucent, screenshot for reference:

![image](https://github.com/Developer-DAO/academy-turbo/assets/2791028/d382f086-3c24-49b1-afbf-9e62daa178a0)

Can we update this to have a solid color and not be transparent? :pray: 
thank you!

Reply:
also, this:

![image](https://github.com/Developer-DAO/academy-turbo/assets/2791028/384bd391-4886-40a4-acd0-91edbbae095d)

So, we want the verify process to be _mandatory_ because this step creates the account, or logins the user, etc.

thank you!

Reply:
I definitely missed this. will update the light mode shortly.

Reply:
clicking on the backdrop usually closes the modal, not sure, but I will read more Rainbowkit docs to see if it can be disabled. 
The login flow will still be incomplete even if the user clicks outside the modal.

I think the authentication status will still be disconnected, hence the connect button will still appear even after the modal is closed.
the user can click on the connect button and resume directly from the verification.

Reply:
Okay, I fixed the modal background issue, but I couldn't work around clicking outside the modal; not closing the verification modal, we can leave that for later.

The modal is dark on dark theme, light on light theme. what do you think?

Reply:
> Okay, I fixed the modal background issue, but I couldn't work around clicking outside the modal; not closing the verification modal, we can leave that for later.
> 
> The modal is dark on dark theme, light on light theme. what do you think?

Alright, sound good mate!  we can leave the clicking outside as an only nice to have, is not too important right now nor mandatory :)

`The modal is dark on dark theme, light on light theme. what do you think?`

Agree! Are you up to add this on this PR also? 
in other case we can create a new task for it, move forward as how you feel comfortable!

Thank you @peteruche21 :rocket:

Reply:
alright resolving this one